core/javax.cache/src/main/java/javax/cache/CacheManager.java

```diff
@@ -127,13 +125,14 @@ public interface CacheManager {
      */
     boolean removeCache(String cacheName);
 
-    /**
-     * This method will return a UserTransaction.
-     *
-     * @return the UserTransaction.
-     * @throws UnsupportedOperationException if JTA is not supported
-     */
-    UserTransaction getUserTransaction();
+    // TODO javax.transaction package is no longer available. Need to find a way to add this back
```

Comment:
`javax.transaction` is not available with JDK 17. Since this seems to be not used, removed. But need to validate the functionality.